lib/phoenix/controller/logger.ex

```diff
@@ -4,6 +4,10 @@ defmodule Phoenix.Controller.Logger do
   @behaviour Plug
   import Phoenix.Controller
 
+  alias Phoenix.Endpoint.Instrument
+
+  @filter_parameters Application.get_env(:phoenix, :filter_parameters)
```

Comment:
I wouldn't have mind fixing it but thanks for doing it @josevalim!

I have two questions:

1. Is there a significant performance penalty with calling `Application.get_env/2` each time compared to using a module attribute?

2. While working on this PR I kept wondering if Phoenix should somehow wrap `Logger` with its own module so that we can ensure no logs are left unfiltered. I can see some downsides with this approach but I was curious what your opinion was.